Need to add more fields to CoinMasterShopRequest that uses ShopRows

Veracity0 opened the pull request:
Which would be just the Kiwi Kwiki Mart, among coinmasters that are currently accessible.

jaadams5 reviewed (dismissed):
>who needs a code review for a fix this simple? :)

:-) indeed.

jaadams5 commented:
Of course I should have waited to see the tests pass...

Veracity0 commented:
> Of course I should have waited to see the tests pass...
Ha ha.

I added ajax=1 - which works fine and makes the responseText much smaller - and the test was assuming no ajax in the urlString.

I fixed the test.
I should change the response text it parses...

jaadams5 reviewed (approved):
Second time's the charm.
